StarRailBattleSim/src/relics/ornament/TaliaKingdomOfBanditry.java

```diff
@@ -0,0 +1,36 @@
+package relics.ornament;
+
+import characters.AbstractCharacter;
+import powers.PermPower;
+import relics.AbstractRelicSetBonus;
+
+public class TaliaKingdomOfBanditry extends AbstractRelicSetBonus {
+    public TaliaKingdomOfBanditry(AbstractCharacter owner, boolean fullSet) {
+        super(owner, fullSet);
+    }
+
+    public TaliaKingdomOfBanditry(AbstractCharacter owner) {
+        super(owner);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(new TaliaKingdomOfBanditryPower());
```

Comment:
the unconditional bonus should be a power in on equip, and the conditional bonus should be a power in onCombatStart